front/components/footer/footer.scss

```diff
@@ -4,7 +4,7 @@
   width: 100%;
   margin-top: 10px;
 
-  .col-md-3,.col-md-2{
+  .col-lg-3{
```

Comment:
you really shouldn't overwrite twitter bootstrap styles, but instead create your own class